frontend/src/scenes/project-homepage/ProjectHomepage.tsx

```diff
@@ -43,7 +44,7 @@ export function ProjectHomepage(): JSX.Element {
                 Invite members
             </LemonButton>
             <LemonButton
-                data-attr="project-home-new-insight"
```

Comment:
I wonder if this is used anywhere in autocapture analytics....

Reply:
This is definitely a mistake on my side, thanks for spotting that!